infoblox/infoblox.py

```diff
@@ -440,13 +413,11 @@ def delete_cname_record(self, fqdn):
         """ Implements IBA REST API call to delete IBA cname record
         :param fqdn: cname in FQDN
         """
-        rest_url = 'https://' + self.iba_host + '/wapi/v' + \
-            self.iba_wapi_version + '/record:cname?name=' + \
-            fqdn + '&view=' + self.iba_dns_view
+        rest_url = "{}/record:cname?name={}&view='{}".format(self.base_url,
```

Comment:
- It looks like there may be a single quote left over in the `view` parameter.
- You can use the `get` kwargs `params` with a dict to uriencode the input strings.